examples/index.js

```diff
@@ -77,6 +77,7 @@ var commitConfig = {
   dotStrokeWidth: 10,
   messageHashDisplay: false,
   messageAuthorDisplay: true,
+  messageShort: "C1",
```

Comment:
I don't think `messageShort` is explicit enough for people without context.

What do you think about something like `commitDotText`? Something else?

Reply:
Great idea. I'll update.